blocks/feed-newsroom/feed-newsroom.css

```diff
@@ -10,6 +10,10 @@
     height: 250px;
   }
 
+main .section.auto-top-spacing.no-margin-top.newsroom .section-container {
```

Comment:
If I understand right, you want to get rid to top margin due to `auto-top-spacing`. There's an exclusion list of `auto-top-spacing` [here](https://github.com/hlxsites/sunstar/blob/7462726403b349ab911ea3f3e9b8419ef4fdb095/scripts/scripts.js#L319). I feel it would be better to add there instead of adding this deep selector.
/cc @dnbute